marquee/marquee.ino

```diff
@@ -766,11 +766,9 @@ void handleConfigure() {
   server.sendContent(form);
 
   form = CHANGE_FORM2;
-  String isPmChecked = "";
-  if (IS_PM) {
-    isPmChecked = "checked='checked'";
-  }
-  form.replace("%IS_PM_CHECKED%", isPmChecked);
+  String isPmOptions = "<option value='NO'>None</option><option value='-1,6'>Lower Right Edge</option><option value='-2,6'>Lower Right Indented</option><option value='-1,0'>Upper Right Edge</option><option value='-2,0'>Upper Right Indented</option><option value='0,6'>Lower Left Edge</option><option value='1,6'>Lower Left Indented</option><option value='0,0'>Upper Left Edge</option><option value='1,0'>Upper Left Indented</option>";
```

Comment:
I have struggled with over customizing the web interface -- right now the code is skirting the edges of the memory limitations and so I struggle with adding items that may or may not be used very much.    I will consider this.